deepchem/utils/grover.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+from typing import List, Any
+from deepchem.feat.graph_data import BatchGraphData
+
+try:
+    import torch
+except ModuleNotFoundError:
+    pass
+
+
+def _get_atom_scopes(graph_index):
+    """Atom scope is a list of tuples with a single entry for every
+    molecule in the batched graph. The entry indicates the beginning
+    node index for a molecule and the number of nodes in the molecule.
```

Comment:
Can you add a `Parameters` section here documenting the types of the argument?

Reply:
added